doc/kernel/services/interrupts.rst

```diff
@@ -608,19 +608,19 @@ connected.
 Going Beyond the Default Supported Number of Interrupts
 -------------------------------------------------------
 
-When generating interrupts in the multilevel configuration, 8-bits per level is the default
+When generating interrupts in the multi-level configuration, 8-bits per level is the default
 mask used when determining which level a given interrupt code belongs to. This can become
 a problem when dealing with CPUs that support more than 255 interrupts per single
 aggregator. In this case it may be desirable to override these defaults and use a custom
 number of bits per level. Regardless of how many bits used for each level, the sum of
 the total bits used between all levels must sum to be less than or equal to 32-bits,
 fitting into a single 32-bit integer. To modify the bit total per level, override the
 default 8 in `Kconfig.multilevel` by setting :kconfig:option:`CONFIG_1ST_LEVEL_INTERRUPT_BITS`
-for the  first level, :kconfig:option:`CONFIG_2ND_LEVEL_INTERRUPT_BITS` for the second tier and
-:kconfig:option:`CONFIG_3RD_LEVEL_INTERRUPT_BITS` for the third tier. These masks control the
+for the  first level, :kconfig:option:`CONFIG_2ND_LEVEL_INTERRUPT_BITS` for the second level and
+:kconfig:option:`CONFIG_3RD_LEVEL_INTERRUPT_BITS` for the third level. These masks control the
 length of the bit masks and shift to apply when generating interrupt values, when checking the
 interrupts level and converting interrupts to a different level. The logic controlling
-this can be found in `irq.h`
+this can be found in `irq_multilevel.h`
```

Comment:
single backquotes render as italics which I don't think is what you want here:

```suggestion
this can be found in :file:`irq_multilevel.h`
```

Reply:
indeed, fixed, and thanks!